docs/03-code-internals/20-form-kit.md

```diff
@@ -1840,3 +1840,13 @@ Allows refusing a field. Only available for: [question](/docs/guides/frontend/fo
 ```ruby
 field.refuse
 ```
+
+### upload_image
+
+Takes an image path on the filesystem and uploads it for the field. Only available for: [image](/docs/guides/frontend/form-kit/controls/image).
```

Comment:
This link doesn't go anywhere... (not your fault, seems like they're all copy/pasted from our docosaurous experiment). 

```suggestion
Takes an image path on the filesystem and uploads it for the field. Only available for the `Image` control.
```